recipe/meta.yaml

```diff
@@ -39,22 +39,22 @@ requirements:
   build:
     - 7za               # [win]
     - bzip2 1.0.*       # [unix]
-    - curl              # [win]
+    - curl >=7.44.0,<8              # [win]
     - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
+    - readline 7.0     # [unix]
+    - sqlite 3.20.*     # [unix]
+    - tk 8.6.*          # [unix]
     - xz 5.2.*          # [unix]
     - zlib 1.2.11        # [unix]
-    - ncurses 5.9*      # [linux]
+    - ncurses 5.9      # [linux]
   run:
     - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
+    - readline 7.0     # [unix]
+    - sqlite 3.20.*     # [unix]
+    - tk 8.6.*          # [unix]
```

Comment:
Even though this was bumped to 8.6, it is still linking against 8.5. Also it appears to now being using the system Tcl/TK, which it [was not doing before]( https://travis-ci.org/conda-forge/python-feedstock/builds/297158425#L7201-L7203 ).

ref: https://github.com/conda-forge/python-feedstock/blob/c2fe3e8753fa50e30b4a27c7735dfd6c0a1dd08a/recipe/build.sh#L35
ref: https://travis-ci.org/ocefpaf/python-feedstock/builds/312990050#L7221-L7222

Reply:
Try to address this in PR ( https://github.com/conda-forge/python-feedstock/pull/162 ).